template/config/nightwatch/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "test:e2e": "nightwatch tests/e2e/*"
+  },
+  "devDependencies": {
+    "chromedriver": "^124.0.3",
+    "geckodriver": "^4.4.0",
+    "nightwatch": "^3.6.2",
+    "ts-node": "^10.9.2",
+    "vite-plugin-nightwatch": "^0.4.6"
+  },
+  "dependencies": {
+    "@vitejs/plugin-vue": "^5.0.5"
```

Comment:
Do we need it?

Reply:
yes we need it because when i remove this project crash